### PR TITLE
feat: Added Lists Component

### PR DESCRIPTION
**Pull request by MaruthiPothuganti**
Added
- list types.
- Stacked list.
- Floated list.

### CODE REVIEW

File: Components/Lists/list.html [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Lists | LightUI</title>
+    <link rel="stylesheet" href="../../Docs/doc.css">
+    <link rel="stylesheet" href="list.css">
+</head>
+
+<body>
+
+    <div class="nav flex-space-between padding-xxl shadow-sm">
+        <!-- <ion-icon class="header-icon" name="sunny-outline"></ion-icon> -->
```
Comment (nerdyspook) [inline Components/Lists/list.html:16]: You can remove this commented code from here.

**Review by nerdyspook — APPROVED** [review]
Code seems good to merge.

File: Components/Lists/list.html (hunk 1 above)
Comment (MaruthiPothuganti) [reply Components/Lists/list.html:16]: Thanks. Actually, this is to switch between dark and day mode. I will be working on it in later changes.

File: Components/Lists/list.css [hunk 2]
```diff
@@ -0,0 +1,66 @@
+@import url(/CSS/variable.css);
+.list-container {
+    gap: 50px;
```
Comment (shivampandey0) [inline Components/Lists/list.css]: `rem` should be used for greater than `10px`

File: Components/Lists/list.css [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+.list,
+.stacked-list {
+    font-size: 1.5rem;
+}
+
+.roman {
+    list-style-type: lower-roman;
+}
+
+.roman {
```
Comment (shivampandey0) [inline Components/Lists/list.css]: duplicate with line no. 11

File: Components/Lists/list.html [hunk 4]
```diff
@@ -0,0 +1,242 @@
...
+            </div>
+
+            <p class="padding-l">To use the above stacked list components, just copy and paste the below code snippets:</p>
+            <iframe src="https://www.thiscodeworks.com/embed/62062617a4f5770015a8853a" style="width: 100%; height: 281px;" frameborder="0"></iframe>
+
+
+            <div class="divider margin-v-l"></div>
+
+            <h2 class="sub-heading">Floated lists</h2>
+            <p class="padding-l">Stacked lists are generally seen in the notifications as given below.</p>
```
Comment (shivampandey0) [inline Components/Lists/list.html]: List description should be for `Floated lists` currently same as `Stacked lists`

**Review by shivampandey0 — COMMENTED** [review]
Code looks good.
Added some suggestions to remove the duplication.

**Comment by MaruthiPothuganti** [comment]
The pr fixes #10

**Pull request merged**